config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-presets.yaml

```diff
@@ -81,3 +91,12 @@ presets:
     value: m3.xlarge
   labels:
     preset-kubernetes-e2e-aws-eks-1-11: "true"
+
+- env:
```

Comment:
It seems like these envs should be common to all EKS jobs? If so, putting them in `preset-kubernetes-e2e-aws-eks-common` may be better.